src/cluster.ts

```diff
@@ -75,7 +76,10 @@ export type GetClustersCallback = (
   apiResponse?: google.bigtable.admin.v2.IListClustersResponse
 ) => void;
 export interface SetClusterMetadataOptions {
-  nodes: number;
+  nodes?: number;
```

Comment:
is this considered breaking if we change something from required to optional? I'm guessing it would be the other way around, but want to confirm :)

Reply:
Here we are changing something from required to optional so that would make it not a breaking change since users can still pass just `nodes` in. This is where we make `nodes` optional because instead the user can provide autoscaling parameters. These parameters will be validated against a new validation function and an error will be thrown if an invalid combination of parameters are provided.

Reply:
I would not consider switching from required to optional for TypeScript breaking 👍